samples/command/consumer/src/main.rs

```diff
@@ -80,59 +78,31 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let addr: SocketAddr = consumer_authority.parse()?;
```

Comment:
Consider having the HTTP Consumer Server setup in a ``samples/common`` since it's the same for all the samples.

Reply:
I don't really want a samples/common folder.  I prefer if each sample contains most of its own code.